biome.json

```diff
@@ -0,0 +1,41 @@
+{
+  "formatter": {
+    "enabled": true,
+    "formatWithErrors": false,
+    "ignore": [],
+    "attributePosition": "auto",
+    "indentStyle": "space",
+    "indentWidth": 4,
+    "lineEnding": "lf",
+    "lineWidth": 88
+  },
+  "linter": {
+    "enabled": true,
+    "rules": {
+      "style": {
+        "useBlockStatements": "warn",
+        "noShoutyConstants": "warn"
+      }
+    }
+  },
+  "javascript": {
+    "formatter": {
+      "arrowParentheses":"always",
+      "bracketSameLine": false,
+      "bracketSpacing": true,
+      "jsxQuoteStyle": "double",
+      "quoteProperties": "asNeeded",
+      "quoteStyle": "single",
+      "semicolons": "asNeeded",
+      "trailingComma": "es5"
+    }
+  },
+  "json": {
+    "formatter": {
+      "trailingCommas": "none"
+    }
+  },
+  "organizeImports": {
+    "enabled": true
+  }
+}
```

Comment:
oooof, that's some configuration, is there no "default, widely accepted as the standard" config, like for prettier/black/ruff...?

I guess i'm miss having a non-configurable "elm-format" like tool around.
Anyway, thanks for telling me about biome, looks nice!

Reply:
For the record, after discussion, that configuration is the new "default, widely accepted as the standard" :p